userguide/sharing.rst

```diff
@@ -920,7 +920,9 @@ provides more details for each configurable option.
    |                                |               |          |                                                                                                                                                      |
    +--------------------------------+---------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------+
    | Export Recycle Bin             | checkbox      | ✓        | Set for deleted files to move to :file:`.recycle` in the root folder of the share. The :file:`.recycle` directory can be deleted to reclaim space    |
-   |                                |               |          | and is recreated whenever a file is deleted.                                                                                                         |
+   |                                |               |          | and is recreated whenever a file is deleted. If a file being deleted is in a subdirectory and larger than 20 MiB, adjust the                         |
```

Comment:
Not clear what the parameter does.  I suspect that files larger than the set size are not kept in .recycle, but just deleted.  Please clarify.